terraform/environments/mlra/locals.tf

```diff
@@ -52,4 +52,6 @@ locals {
   # example usage:
   # example_data = local.application_data.accounts[local.environment].example_var
   application_data = fileexists("./application_variables.json") ? jsondecode(file("./application_variables.json")) : {}
+
+  cluster_name = "mlra-ecs-cluster"
```

Comment:
A bit more subjective, but not sure about having this value in here. Perhaps something like app_ecs_cluster = "${local.application_name}-ecs-cluster" in the module call would be cleaner

Reply:
updated this